### DISCUSSION TITLE
Any idea why forward proj are all black

### DISCUSSION THREAD

**Discussion by lhr522857918** [CERN/TIGRE#455] [Q&A]
Thanks so much for this package!
![image](https://github.com/CERN/TIGRE/assets/118142275/d01b9c60-afc5-4cf4-bb30-1ed7f9287a36)
This is my geo information. and I am using a head phantom.
After forward projection, the output projections are totally black.

**Comment by AnderBiguri** [comment]
Hi @lhr522857918 , can you share this in text form (code, not an image) so I can try it?
